src/main/java/com/google/devtools/build/lib/actions/SpawnResult.java

```diff
@@ -213,6 +213,9 @@ String getDetailMessage(
   /** Returns a file path to the action metadata log. */
   Optional<MetadataLog> getActionMetadataLog();
 
+  /** Whether the spawn result was obtained through remote strategy. */
```

Comment:
s/remote strategy/remote execution/

Reply:
strategy was intentional here - both pure remote cache and remote execution with their independent RemoteModule configuration emit true here - any SpawnResults created under the `build/lib/remote` package will have this set.